features/dd-sdk-android-rum/src/main/kotlin/com/datadog/android/rum/internal/metric/SessionEndedMetric.kt

```diff
@@ -59,10 +65,14 @@ internal class SessionEndedMetric(
         wasStopped = true
     }
 
-    fun toMetricAttributes(): Map<String, Any?> {
+    fun onMissedEventTracked(missedEventType: MissedEventType) {
+        missedEventCountByType[missedEventType] = (missedEventCountByType[missedEventType] ?: 0) + 1
+    }
+
+    fun toMetricAttributes(ntpOffsetAtEnd: Long): Map<String, Any?> {
```

Comment:
it is better to specify the unit to avoid bugs

```suggestion
    fun toMetricAttributes(ntpOffsetAtEndMs: Long): Map<String, Any?> {
```

Reply:
Updated all the related param names.